### DISCUSSION TITLE
Benefits are not available yet

### DISCUSSION THREAD

**Discussion by jaedm97** [community/community#141859] [GitHub Education]
Hi,

It's been almost 72hours, since my application being approved. But still I don't get the academic benefits.

Can anyone please help me?

Thank you

**Comment by davevad93** [comment]
Hi @jaedm97 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @jaedm97 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
